.gitignore

```diff
@@ -5,7 +5,6 @@ Desktop
 Documents
 Downloads
 Dropbox
-Library
```

Comment:
Realized this folder should not be ignored because I already use it to store ST3 settings.